### PR TITLE
paging

### PR DESCRIPTION
**Pull request by Omid07**

### CODE REVIEW

File: app/src/main/java/com/example/omid/rss_5/NewsActivity.java [hunk 1]
```diff
@@ -65,22 +66,68 @@ protected void onCreate(Bundle savedInstanceState) {
...
-        for (int i = 0; i < categoryNewsList.size(); i++) {
-            mCategoryNewsesList.add(i, categoryNewsList.get(i));
+        mNewsItemShow =categoryNewsList;
+        for (int i = 0; i < Constant.LOAD_ITEM && i < categoryNewsList.size(); i++) {
+            mCategoryNewsesList.add(categoryNewsList.get(i));
         }
-        mAdapter = new CategoryNewsAdapter(categoryNewsList);
+        mAdapter = new CategoryNewsAdapter(mCategoryNewsesList);
         mRecyclerView.setAdapter(mAdapter);
+        mRecyclerView.addOnScrollListener(new RecyclerView.OnScrollListener() {
```
Comment (nvmanh) [inline app/src/main/java/com/example/omid/rss_5/NewsActivity.java]: duplicated with line 109, please make 1 method and reuse

File: app/src/main/java/com/example/omid/rss_5/NewsActivity.java [hunk 2]
```diff
@@ -65,22 +66,26 @@ protected void onCreate(Bundle savedInstanceState) {
...
     public void showData(ArrayList<CategoryNews> categoryNewsList) {
         mCategoryNewsesList = new ArrayList<>();
-        for (int i = 0; i < categoryNewsList.size(); i++) {
-            mCategoryNewsesList.add(i, categoryNewsList.get(i));
+        mNewsItemShow =categoryNewsList;
+        for (int i = 0; i < Constant.LOAD_ITEM && i < categoryNewsList.size(); i++) {
+            mCategoryNewsesList.add(categoryNewsList.get(i));
         }
-        mAdapter = new CategoryNewsAdapter(categoryNewsList);
+        mAdapter = new CategoryNewsAdapter(mCategoryNewsesList);
```
Comment (nvmanh) [inline app/src/main/java/com/example/omid/rss_5/NewsActivity.java]: please implement this activity RecyclerView.OnScrollListener, then mRecyclerView.addOnScrollListener(this)

Comment (Omid07) [inline app/src/main/java/com/example/omid/rss_5/NewsActivity.java]: @nvmanh https://developer.android.com/reference/android/support/v7/widget/RecyclerView.OnScrollListener.html
now the RecyclerView.OnScrollListener is an  abstract class . so i can not implement this. also i can not extend it from this activity as i have already extends  AppCompatActivity.. what should i do now ??

**Comment by nvmanh** [comment]
yeah! not update newest code, it's my mistake

**Comment by nvmanh** [comment]
OK!

**Pull request merged**